brandingbrand/flagship
# refactor(fscomponents): FLAGSHIP-106 Adding ADA to component

**Ticket:**
  - https://jira.brandingbrand.com/browse/FLAGSHIP-106
## **Description:**
  - Adding accessibility to CartCount component
## **Test:**
  - run fs tests
  - confirm with screen reader
